### DISCUSSION TITLE
录入感悟

### DISCUSSION THREAD

**Discussion by PirateBook1** [banned-historical-archives/banned-historical-archives.github.io#2095] [General]
[纪念中国共产党成立一百周年](https://banned-historical-archives.github.io/articles/53df6db6c1/)

一些感悟：

> 秘鲁共产党和贡萨罗主席总结了毛泽东思想的普遍贡献，提出了无产阶级意识形态的最高阶段——马克思列宁毛主义。

这里“最高阶段”的提法不赞同，“当今无产阶级意识形态的最高阶段”比较妥当。是不是从当前一直到共产主义实现用的都是毛主义，就没有新的理论进步了？一些问题如在帝国主义国家的持久人民战争（美帝、中帝）究竟怎么打、文化大革命中的新理论究竟在今后的文化革命中有多少是普适的，当前实际上并没有搞清楚。

> 毛主席对资产阶级的攻击进行了反击，命令所有军队停止镇压造反派，释放被捕群众。保守派军事领导人不得不执行毛主席的指令。**然而，保守派军事领导人并没有失去他们的权力，他们利用军队的力量重建在前一时期被瓦解的保守派群众组织，对抗造反派群众组织。**

> 为了结束群众组织之间愈演愈烈的内战，毛主席命令人民解放军再次介入运动，协助革命群众组织结成联盟，建立稳固的新政权。各群众组织进行了整合，原来的派系解散了。

这步棋在我看来是中央的错棋，对军头太过纵容。有相当大一部分军队在敌人手里的情况下，动用军队支持左派，到底对运动实质起了镇压还是促进作用是很值得怀疑的。中央还提出“大联合”，这种“联合”出来的革委会能有多少质量也是值得打一个问号的。

> 但是，党内高层资产阶级分子并没有被彻底清算，他们抵制江青、张春桥等作为无产阶级革命者的同志，支持邓小平复出。
> 
> 1975年，邓小平发起“整风”运动，针对革命造反派和文化大革命中的新生事物，许多造反派被镇压。毛主席拯救邓小平的努力的徒劳的。

把邓扶上来，中央的责任难辞其咎。

> 1976年，毛主席发起“批邓、反击右倾翻案风”运动，邓小平和许多其他右派分子被撤职。由于需要稳定局势，采取中间派立场的华国锋取代了邓小平。

“由于需要稳定局势”这个理由很难赞同。

> 收到消息后，上海的左派领导人打算发动起义，但由于领导人的软弱无能〔6〕，最终没有实现。

上海的问题注释已经说过了。再放眼全国，政变发生后为什么左派毫无招架之力，恐怕还是得从社会基础层面分析。敌人已经不能说是“一小撮”了，毛主席也说恐怕是“相当大一部分不在马克思主义者手里”。**广大工厂、公社实际上没有实现公有制的本质——人民当家作主。**走资派广泛把持地方政权，在地方实际上早已行的是资产阶级专政之实。走后门、走关系等等各种资产阶级法权实际上是没有得到有效遏制的。

话说回来，许多年过去，左派好像并没有系统总结毛错误的理论文章。看事物我们得看主流和支流，毛主席的伟大地位是不容置疑的，但是毛是不是没有错的圣人？正如列宁分析马克思、毛评价斯大林一样，毛主席自然也是功过一体两面的。

@banned-historical-archives @CiangCing14

**Comment by ghost** [comment]
華國鋒絕對不是中間派，就是極右派。小組之所以現在沒了消息，是因為這種統一戰線是不牢靠的，是不確定的，是不穩定的，正像所有實用主義者一樣，他們還需要成長。但對華國鋒的歷史定位不可退讓一步。最後別搞成下一個臺爾曼就好。

**Reply by ghost** [reply]
陳X忠反正是已經成泰爾曼了，中文小組也是差不多。別又有什麼新人又因為立場不夠堅定而進去了，再讓毛澤東思想衝擊一遍馬列毛主義網站。

**Comment by ghost** [comment]
PAPER編輯組有一個專門給“實幹小組”看的“共識綱領”，並且是只有實幹小組才能看，這個可以回答你的中帝怎麼打的問題，但是不幸的是，我和PAPER編輯組和後來人發生了冷戰，所以也就沒有獲知這份綱領的權限，此外就是PAPER已經放棄了資產階級軍事戰術，關閉MEGA網盤地址了。

**Reply by ghost** [reply]
看PAPER的內部刊物，沒有此綱領的影子，只是提及了“自我定位的渴望”和“地下宗派活動”還有“逃離新疆當局虎口”等等。

**Reply by PirateBook1** [reply]
PAPER 内部有很严重的机会主义，还有理论上未能充分接受毛主义。有一期贡萨罗主席的新闻他们甚至把名字打错了。

**Reply by ghost** [reply]
> PAPER 内部有很严重的机会主义，还有理论上未能充分接受毛主义。有一期贡萨罗主席的新闻他们甚至把名字打错了。

你這麼一說我確實想起來這件事了，謝謝提醒。

**Comment by ghost** [comment]
二月逆流的中央根源在於毛澤東本人於一月風暴期間讓葉劍英拍傷手骨有關（京西賓館事件），要當心中央出修正主義。

**Comment by ghost** [comment]
提及“毛澤東思想”一詞的由馬列毛主義者撰寫的文章都是系統批判毛的錯誤的文章，這不是很明顯嗎？

**Comment by ghost** [comment]
我们将要做什么？
我们的目标是什么？
（出于安全起见，以上两内容不向除实干小组以外的群体、小组开放，请各实干小组主动联络索取未删减版）

**Comment by PirateBook1** [comment]
> 为什么文化大革命最后会失败？我们认为，主要原因是没有充分运用《以人民战争直至共产主义来庆祝无产阶级文化大革命五十周年！》中指出的“武装群众的汪洋大海”的无产阶级军事路线，这本来可以通过赋予人民民兵更多的权力来实现。1967年，毛主席指示张春桥和王洪文在上海扩大民兵，反对走资派，但这个方针没有在全国实现。高层左派领导人被偷袭逮捕后，支持在全国继续革命的干部和群众陷入无力的境地，没有领导，没有组织，没有武器，使他们无法对抗资产阶级分子的镇压。

文革为什么失败？根本原因还是得从经济上找，归因于军事路线没有触及本质。主要原因是资产阶级法权没有得到限制反而被扩大了，社会主义生产关系被广泛篡夺了领导权的走资派侵蚀。

广大工农在这种被侵蚀的生产关系下生活水平增长缓慢，复辟后为什么左派毫无还手之力，政权的变更为什么能相对稳定——人民逐渐能吃饱穿暖了，能吃上肉了，这种最直接的、看得见摸得着的利益使得广大工农在复辟一开始时站到了资产阶级这边（对左派的镇压，揭批查等当然是不能忽视的原因）——当然，生产力的迅速发展是基于社会主义时期的工业基础，基于被完全放开的发展资本主义限制，基于外国资本的大量进入，基于剥削关系的重新确立。

限制资产阶级法权，在实际上确立人民当家作主是重中之重。

@CiangCing14

**Reply by ghost** [reply]
現在有一種普遍不贊成出動軍隊鎮壓四五事件的傾向，也許動用軍隊對付反革命暴亂有些強化資產階級國家傾向，就在反擊右傾翻案風裡，中文小組走了官方文革的老路，用資產階級國家限制資產階級法權，於是起到了失敗的效果。

**Reply by ghost** [reply]
華XX法西斯主義復辟多半與當年毛和中文小組用軍隊鎮壓“異端派”引起的省份混亂給華XX留下的陰影記憶有關，也就是說，華XX的中國大陸大屠殺（水晶之夜）本質上是歪曲、聯合楊XX、省XX對貢薩羅主席、省XX、毛和中文小組的一次總復仇主義。

**Reply by ghost** [reply]
我認為反擊右傾翻案風時期社會的主要矛盾是腦體矛盾，正如《關於科技發展的若干問題》一樣，問題在於使人民群眾真正懂得三大差別，最後用革命暴力解決矛盾，當時就應該提出重建戰鬥隊和普及武器製作知識等等的宣傳任務，而經濟方面則應該放寬武器貿易管控。然而，人民逐漸能吃飽似乎是一個偽命題，人民在所謂“吃不飽”的情況下都能製作出原子彈、衛星、可控核聚變，而王洪文也算人民，王洪文的生活水平與當代中國人的生活水平差不多；所謂“吃不飽”純粹是拿自然災害期間無限擴展到整個毛澤東時代的結果；而實際上反擊右傾翻案風期間中國人民的生活水平實際上應該比現在還高。（根據揭發王洪文同志的材料。）王洪文同志當時吃大閘蟹，江青同志吃奶油蛋糕，可見所謂“吃不飽”完全是華XX法西斯主義靠飢餓維持的資本主義秩序的描述。

**Reply by ghost** [reply]
一月風暴期間的上海人民公社食堂的影片在YouTube上流出了，從其中可以看到，一月風暴期間人民群眾就已經能吃上肉餃子了。

**Reply by ghost** [reply]
我認為中國生產力根本沒有多大發展，純粹是矮個子裡挑高個子，人為製造大量飢餓，然後片面地進行“經濟宣傳”的結果。現在的所謂航天技術和核武器的發展，純粹是寅吃卯粮。並且還都是瘸腿的人，反擊右傾翻案風時期的經濟，根據俄羅斯學者的研究，足以撐起登月計劃和登陸火星的計劃，然而，某些人的眼光比他的個子還要低。

**Reply by ghost** [reply]
世界上最不能發展生產力的上層建築就是“唯生產力論”。

**Reply by ghost** [reply]
資產階級可以靠收買、福利主義僱傭自己的私人軍隊（一種比工人貴族更差勁的群體）；所謂“看得見、摸得著”的利益，純粹是挖人民解放軍牆角，然後黨內資產階級組織的職業合同制軍隊罷了。而所謂的“廣大工農群眾”，則不能解釋“無產階級文化大革命編年史”中接著“毛主席紀念堂”之後的“全國大武鬥”是怎麼回事，可見當時早就有全國範圍的大罷工了。政權根本沒有陷入穩定。

**Reply by PirateBook1** [reply]
你说得对，上海文化革命推进得相对于我国其他地区要好得多。但是我国造反派夺权人民当家作主的地区毕竟是少的，上海毕竟不能代表主流。

**Reply by PirateBook1** [reply]
只有上海、辽宁、吉林、黑龙江、云南、安徽说得上造派控制局面或主政。

**Reply by ghost** [reply]
真要從經濟上找，是資產階級的國家資本主義惡性膨脹（而不是無產階級的國家資本主義，如“收權治散”沒有得到有效抑制）、擴軍主義的盛行導致的資產階級法權的擴大（見一打三反時期，不是說“政治是不流血的戰爭”嗎？）、以及乒乓外交帶來的一系列外交醜聞（喪權辱國的美國核保護傘事件）、以及宣傳部對拉美馬列毛主義革命、中央文革文藝的忽視的認識問題（即所謂“無產階級文化大革命”的官方結束）以及沒有爭取到大量生產隊農民支持的結果。

**Reply by PirateBook1** [reply]
> 而王洪文也算人民，王洪文的生活水平與當代中國人的生活水平差不多
> 反擊右傾翻案風期間中國人民的生活水平實際上應該比現在還高。（根據揭發王洪文同志的材料。）王洪文同志當時吃大閘蟹，江青同志吃奶油蛋糕

王和江两位同志的生活也不能完全排除资产阶级法权。虽然从各种材料来看他们在极力避免

**Reply by PirateBook1** [reply]
> 真要從經濟上找，是資產階級的國家資本主義惡性膨脹（而不是無產階級的國家資本主義，如“收權治散”沒有得到有效抑制）、擴軍主義的盛行導致的資產階級法權的擴大（見一打三反時期，不是說“政治是不流血的戰爭”嗎？）、以及乒乓外交帶來的一系列外交醜聞（喪權辱國的美國核保護傘事件）、以及宣傳部對拉美馬列毛主義革命、中央文革文藝的忽視的認識問題（即所謂“無產階級文化大革命”的官方結束）以及沒有爭取到大量生產隊農民支持的結果。

这段说得好，你的理解要比我深刻

**Reply by ghost** [reply]
你看看機會主義者、柬埔寨的華XX——波XXX的狂妄自大，就知道外債規模有多龐大了。

**Reply by ghost** [reply]
消滅本國貨幣不代表消滅貨幣。

**Reply by PirateBook1** [reply]
@CiangCing14 你这个指的是哪本资料？《無產階級文化大革命編年史》

**Reply by ghost** [reply]
> @CiangCing14 你这个指的是哪本资料？《無產階級文化大革命編年史》

https://zh.wikipedia.org/zh-tw/Template:%E6%96%87%E9%9D%A9%E7%BC%96%E5%B9%B4%E5%8F%B2

**Reply by ghost** [reply]
> @CiangCing14 你这个指的是哪本资料？《無產階級文化大革命編年史》

這個模板沒有人提出討論異議，表明修正主義者掩蓋的資料太多太多了。

**Reply by ghost** [reply]
> @CiangCing14 你这个指的是哪本资料？《無產階級文化大革命編年史》

似乎這一部分的“全國大武鬥”包括工農解放軍幾乎被特色和美帝全網封殺。

**Reply by ghost** [reply]
> @CiangCing14 你这个指的是哪本资料？《無產階級文化大革命編年史》

https://zh.wikipedia.org/w/index.php?title=Template%3A%E6%96%87%E9%9D%A9%E7%BC%96%E5%B9%B4%E5%8F%B2&diff=18383507&oldid=18364506

**Reply by ghost** [reply]
> @CiangCing14 你这个指的是哪本资料？《無產階級文化大革命編年史》

https://zh.wikipedia.org/zh-tw/Special:%E7%94%A8%E6%88%B7%E8%B4%A1%E7%8C%AE/74.115.0.2

**Reply by ghost** [reply]
> @CiangCing14 你这个指的是哪本资料？《無產階級文化大革命編年史》

http://www.tywiki.com/index.php/%E4%BB%8E%E5%A4%96%E5%AA%92%E7%9C%8B1976%E5%B9%B410%E6%9C%88%E6%94%BF%E5%8F%98%E5%90%8E%E7%9A%84%E9%98%B6%E7%BA%A7%E6%96%97%E4%BA%89

**Reply by ghost** [reply]
> @CiangCing14 你这个指的是哪本资料？《無產階級文化大革命編年史》

Fire-and-Ice：这个模板列的武斗事件应该都有根据，只有“镇压新疆生产建设兵团兵变”我不知道是怎么回事

**Comment by lance111l** [comment]
社会主义是全世界新的实践   有两面性  关键是有自我矫正能力  总体方向正确就可行   
文化大革命是政治运动 有矫枉过正 可以理解

**Reply by banned-historical-archives** [reply]
你说的总体正确的方向是不是官僚资本主义道路？

**Reply by PirateBook1** [reply]
必须沿着马恩列斯毛的道路

**Reply by lance111l** [reply]
总体方向是为了国民和国家的富裕和强大   那是中国特色的社会主义     别局限在官僚资本主义

**Reply by PirateBook1** [reply]
总体方向是炸毁私有制，实现无产阶级的解放亦即阶级的消灭

**Reply by lance111l** [reply]
hi  piratebook1    do not be silly     only you care about class in China

**Reply by banned-historical-archives** [reply]
> 总体方向是为了国民和国家的富裕和强大 那是中国特色的社会主义 别局限在官僚资本主义

是不是社会主义看看工人地位就知道了
封建套个社会主义的皮就把你们粉红忽悠瘸了

**Reply by lance111l** [reply]
hi banned-historical-archives      管理国家 靠工人的素质行吗  学术有专攻      
你对社会主义的理解狭隘了  别以为能从落魄的英国那里能学到一点点精髓

**Reply by PirateBook1** [reply]
道不同不相为谋，各位不必多言了
